curl-to-elisp.el

```diff
@@ -285,6 +291,29 @@ Return nil if S does not contain CH."
    command
    (rx bos (* blank) (? (in "$#")) (* blank))))
 
+
+;;; ###autoload
+(defun curl-to-elisp-httpie-to-elisp (command)
+  "Convert httpie/curlie COMMAND to Emacs Lisp expression, return the expression.
+
+When called interactively, also pretty-print the expression in echo area."
+  (interactive "shttpie command: ")
+  (if curl-to-elisp-curlie-binary
+      (let ((command (replace-regexp-in-string
+                      "^\\(curlie\\|http\\) "
+                      (format "%s --curl " curl-to-elisp-curlie-binary)
+                      command)))
+        (curl-to-elisp
+         (with-temp-buffer
+           (start-process-shell-command
+            "curl-to-elisp-httpie"
+            (current-buffer)
+            command)
+           (sleep-for 0.1)
```

Comment:
I tried `accept-process-output` to replace sleep but it didn't work well. When with-temp-buffer ends, the process is not killed, so the minibuffer asks me if I want to kill the process.

```el
(with-temp-buffer
  (accept-process-output
   (start-process-shell-command
    "curlie"
    (current-buffer)
    "curlie --curl example.com"))
  (buffer-string))
```

Reply:
I'll give it another shot this weekend and see if I can use `accept-process-output` to avoid the sleep.  ALthough I tested it and get the same results as you.

Usually those messages come from async-shell commands bound to buffers, but running inside a w-t-b should get it right anyway.  I quickly tried fiddling with `async-shell-command-buffer` var to no avail.

/shrug

Reply:
Some more investigation:

As you say, accept-process-output blocks the buffer anyway. It seems that some commands block every run (curlie, or "echo foo"), and some block only sometimes ("ls").

Here's something I found: 

I added a (sleep-for 0) after accept-process-output and it magically worked. Looking at sleep-for's signature, there's an optional ndisp param. When passing `t` there, the block is back.

So, is the redisplay the trick?  It seems so! 

I'm pushing a commit that uses accept-process-output and just `(redisplay)`, and it seems to work fine!